pkg/doc/did/doc_test.go

```diff
@@ -804,6 +806,47 @@ func TestJSONConversion(t *testing.T) {
 	}
 }
 
+func TestJSONWebKey(t *testing.T) {
```

Comment:
Here is an example with DID Doc & JSON Web Key creation. @sudeshrshetty @rolsonquadras @troyronda